cmd/ion-go/main.go

```diff
@@ -65,7 +64,7 @@ func printVersion() {
 		w.FieldName("version")
 		w.WriteString(internal.GitCommit)
 
-		buildtime, err := time.Parse(time.RFC3339, internal.BuildTime)
+		buildtime, err := ion.NewTimestampFromStr(internal.BuildTime, ion.Second, ion.UTC)
```

Comment:
Can we namespace the timestamp-related values more narrowly somehow? `ion.Second` and `ion.UTC` feel out of place at the top-level of an `ion` package to me.

Reply:
Addressed.